scss/_custom-variables.scss

```
@@ -100,6 +100,18 @@ $colors-transparent: map-merge(
 );
 
 
+// >> Gradient Colors
+$gradient-black: linear-gradient(180deg, transparent, $black);
+
+$colors-gradient: ();
+$colors-gradient: map-merge(
+  (
+    "gradient-black":   $black,
```

The linting checks are now failing on the `fusv` (find unused sass variables) step:
```
Looking for unused variables
Searching for unused variables in "/arizona-bootstrap-src/site/assets/scss" folder, scss files...
10 total variables.
No unused variables found in "/arizona-bootstrap-src/site/assets/scss!"
Searching for unused variables in "/arizona-bootstrap-src/scss" folder, scss files...
145 total variables.
1 is not used!
Variable $gradient-black is not being used! /arizona-bootstrap-src/scss/_custom-variables.scss:104
Error: Found 1 unused variable in "/arizona-bootstrap-src/scss" folder
```

```suggestion
    "gradient-black": $gradient-black,
```